src/eko/evolution_operator/__init__.py

```diff
@@ -179,48 +254,130 @@ def quad_ker(
     integrand = ker_base.integrand(areas)
     if integrand == 0.0:
         return 0.0
-
-    # compute the actual evolution kernel
-    if ker_base.is_singlet:
-        gamma_singlet = ad.gamma_singlet(order, ker_base.n, nf)
-        # scale var exponentiated is directly applied on gamma
-        if sv_mode == sv.Modes.exponentiated:
-            gamma_singlet = sv.exponentiated.gamma_variation(
-                gamma_singlet, order, nf, L
+    # TODO refactor calls
+    if order[1] == 0:
```

Comment:
Maybe also here it's easier to split the `quad_ker` in two different functions. All the cals have to be repeated, but they don't share much.